api-reference/10 UI Widgets/dxTreeList/1 Configuration/onEditorPreparing.md

```diff
@@ -12,8 +12,8 @@ EventForAction: dxTreeList.editorPreparing
 Information about the event that caused the function's execution.
 
 ##### field(e.cancel): Boolean
-Allows you to cancel creating the editor.        
-Set it to **true** and implement a custom editor.
+Allows you to cancel the editor creation.        
+Set this field's value to **true** and implement a custom editor.
```

Comment:
You can use inheritance. An example can be found [here](https://raw.githubusercontent.com/DevExpress/devextreme-documentation/c45c9cb15dddf697d35648ca56ca6c422da39f41/api-reference/10%20UI%20Widgets/dxScheduler/3%20Methods/showAppointmentPopup(appointmentData_createNewAppointment_currentAppointmentData).md).